models/tasks.js

```diff
@@ -253,7 +253,8 @@ const fetchUserTasks = async (username, statuses = [], field, order) => {
           .where("participants", "array-contains", userId)
           .orderBy(field, order)
           .get();
-        featureTasksSnapshot = await tasksModel.where("assignee", "==", userId).orderBy(field, order).get();
+        featureTasksSnapshot = await tasksModel.where("assignee", "==", userId).get();
```

Comment:
rather than changing here I suggest we do it here https://github.com/Real-Dev-Squad/website-backend/blob/7e1fe072f031556f64b65320dbb53c4a769ba3da/models/tasks.js#L359. if we don't pass `field` it will go to the else loop

Reply:
done